src/i18n/config.json

```diff
@@ -140,25 +140,25 @@
     "name": "Ukrainian",
     "langDir": "ltr",
     "dateFormat": "DD.MM.YYYY",
-    "hrefLang": "",
+    "hrefLang": "uk",
     "enabled": false
   },
   {
     "code": "zh-cn",
     "localName": "简体中文",
-    "name": "Simplified Chinese",
+    "name": "Chinese",
     "langDir": "ltr",
-    "dateFormat": "DD.MM.YYYY",
-    "hrefLang": "",
+    "dateFormat": "YYYY.MM.DD",
+    "hrefLang": "zh-Hant",
```

Comment:
Sorry to interrupt, but I'm afraid there's a tiny mistake 😂

The `t` in `zh-Hant` means "traditional" while `s` in `zh-Hans` means "Simplified".

Reply:
I just wrote a possible fix, hoping this might help. 😄 https://github.com/nodejs/nodejs.dev/compare/main...zedifen:i18n-ch-fix

Reply:
Oh, gotcha! Could you open a PR? 🙈

Reply:
Sorry, I got a little confused 😅.

Reply:
> Oh, gotcha! Could you open a PR? 🙈

Ok, I've opened a PR. 😃

> Sorry, I got a little confused 😅.

Oh, it's alright 😆 There are lots of variants of Chinese, and their codes do look quite similar